flow/flow.proto

```diff
@@ -89,6 +89,11 @@ message TCPMetric {
   int64 BAFinStart = 6;
   int64 ABRstStart = 7;
   int64 BARstStart = 8;
+
+  int64 IPFragments = 9;
+  int64 TCPSegmentOutOfOrder = 10;
+  int64 TCPRestransmissionPackets = 11;
+  int64 TCPRestransmissionBytes = 12;
 }
```

Comment:
Why do we add fields that currently are not calculated?
I mean Retransmitting, OutOfOrder and so on? I think it is better to add them only when we do implement this part